README.md

```diff
@@ -46,18 +48,15 @@ When installing the given `composer.json` some tasks are taken care of:
 
 ## Updating Drupal Core
 
-Updating Drupal core is a two-step process.
+This project will attempt to keep all of your Drupal Core files up-to-date; the project [drupal-composer/drupal-scaffold](https://github.com/drupal-composer/drupal-scaffold) is used to ensure that your scaffold files are updated every time drupal/core is updated. If you customize any of the "scaffolding" files (commonly .htaccess), you may need to merge conflicts if any of your modfied files are updated in a new release of Drupal core.
+
+Follow the steps below to update your core files.
 
-1. Update the version number of `drupal/core` in `composer.json`.
 1. Run `composer update drupal/core`.
-1. Run `./scripts/drupal/update-scaffold [drush-version-spec]` to update files
-   in the `web` directory, where `drush-version-spec` is an optional identifier
-   acceptable to Drush, e.g. `drupal-8.0.x` or `drupal-8.1.x`, corresponding to
-   the version you specified in `composer.json`. (Defaults to `drupal-8`, the
-   latest stable release.) Review the files for any changes and restore any
-   customizations to `.htaccess` or `robots.txt`.
+1. Run `git diff` to determine if any of the scaffolding files have changed. Review the files for any changes and restore any customizations to `.htaccess` or `robots.txt`.
 1. Commit everything all together in a single commit, so `web` will remain in
    sync with the `core` when checking out branches or running `git bisect`.
+1. In the event that there are non-trivial conflicts in step 2, you may wish to perform these steps on a branch, and use `git merge` to combine the updated core files with your customized files. This facilitates the use of a [three-way merge tool such as kdiff3](http://www.gitshah.com/2010/12/how-to-setup-kdiff-as-diff-tool-for-git.html). This setup is not necessary if your changes are simple; keeping all of your modifications at the beginning or end of the file is a good strategy to keep merges easy.
```

Comment:
Can we stick to 80 column width for the running text in the README? I'm asking this out of pure selfishness, then I can still read it in a half screen width terminal on my laptop ;)